skyrim-platform/src/platform_se/CMakeLists.txt

```diff
@@ -98,26 +98,25 @@ if(NOT "${SKIP_SKYRIM_PLATFORM_BUILDING}")
 
   include(${cmake_dir}/npm.cmake)
 
-  set(wdir "${CMAKE_CURRENT_BINARY_DIR}/_codegen")
-  file(MAKE_DIRECTORY ${wdir})
-  npm_execute_command(
-    WORKING_DIRECTORY ${wdir}
-    COMMAND init -y
-  )
-  npm_execute_command(
-    WORKING_DIRECTORY ${wdir}
-    COMMAND install tsc-watch@4.2.3 typescript@3.8.3 --save
-  )
-
-  npm_execute_command(
-    WORKING_DIRECTORY "${PROJECT_SOURCE_DIR}/tools/dev_service"
-    COMMAND install
-  )
+   set(wdir "${CMAKE_CURRENT_BINARY_DIR}/_codegen")
+   file(MAKE_DIRECTORY ${wdir})
+   npm_execute_command(
+     WORKING_DIRECTORY ${wdir}
+     COMMAND init -y
+   )
+   npm_execute_command(
+     WORKING_DIRECTORY ${wdir}
+     COMMAND install tsc-watch@4.2.3 typescript@3.8.3 --save
+   )
+    npm_execute_command(
+     WORKING_DIRECTORY "${PROJECT_SOURCE_DIR}/tools/dev_service"
+     COMMAND install
+   )
 
   file(GLOB_RECURSE platform_src "skyrim_platform/*")
   list(APPEND platform_src "${CMAKE_SOURCE_DIR}/.clang-format")
-  list(APPEND platform_src "codegen/index.js")
-  list(APPEND platform_src "codegen/FunctionsDump.txt")
+  # list(APPEND platform_src "codegen/index.js")
```

Comment:
**Remove code instead of turning it into comments** - Let Git manage history for you instead.